.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+## What is the purpose of this pull request?
+
+- [ ] Bug fix
+- [ ] New feature
+- [ ] Typo fix
+- [ ] Other, please explain:
+
+## Check if your issue is already reported.
+
```

Comment:
I think this section is actually not needed. PRs could repeat as two people might send a PR for the same thing.